templater/jobs/utils/utils.go

```diff
@@ -18,6 +18,7 @@ var releaseBranches = []string{
 	"1-21",
 	"1-22",
 	"1-23",
+	"1-24",
```

Comment:
having this templater seems like a blessing

Reply:
It is! Thanks you, @abhay-krishna 🙌